cms/check_number_of_missing_files_per_rse

```diff
@@ -58,11 +74,12 @@ if __name__ == '__main__':
         .filter(models.ReplicationRule.state == RuleState.REPLICATING).group_by(models.RSE.rse))
         results = query.all()
         print('Number of replicating files per RSE')
-        for rse, num in result:
-            monitor.record_gauge(stat='judge.number_of_replicating_files.%s' % rse, value=num)
-            print(rse,num)
+        for rse, num in results:
+            replicating_files_gauge.labels(rse=rse).set(num)
+            #monitor.record_gauge(stat='judge.number_of_replicating_files.%s' % rse, value=num)
```

Comment:
For now let's leave both stats and prometheus but add an issue to remove the statsd in a timely way. You can fix this just by changing stat to name.